src/styles/utils/_vars.scss

```diff
@@ -1 +1,11 @@
-$c-gray: #eee;
+$desktop-columns: 12;
```

Comment:
Consider renaming $desktop-columns and $tablet-columns to $desktop-grid-columns and $tablet-grid-columns for better readability and understanding of their purpose.